lib/DiskSpdParser.pm

```diff
@@ -71,53 +201,32 @@ sub parse_cmd_line($$)
     $cmd_line =~ /-o(\d+)/;
     my $ios_per_thread = $1;
     
-    $stats_ref->{'QD'} = $ios_per_thread * $num_threads;
+    $target_stats{'Total'}{'QD'} = $ios_per_thread * $num_threads;
        
-    if( $cmd_line =~ /-Z.*?([^\\]*)\.bin/ )
+    #search for the format of Storscore's pre-generated entropy files
+    if( $cmd_line =~ /-Z.*?([^\\]*)_pct_comp\.bin/ )
```

Comment:
I guess maybe you're intentionally breaking backward compatibility with really old results directories here.  It's probably OK, but you should be aware of it.  A lot of of things in the parser like this were done in order to permit backward compatibility with results obtained before some storscore.cmd change was made.